include/spirv-tools/optimizer.hpp

```diff
@@ -101,10 +101,17 @@ class Optimizer {
   // from time to time.
   Optimizer& RegisterSizePasses();
 
-  // Registers passes that have been prescribed for WebGPU environments.
+  // Registers passes that have been prescribed for converting from Vulkan to
+  // WebGPU.
   // This sequence of passes is subject to constant review and will change
   // from time to time.
-  Optimizer& RegisterWebGPUPasses();
+  Optimizer& RegisterVulkanToWebGPUPasses();
```

Comment:
This is a change to the public interface?  Are we sure nothing depends on this yet?

Reply:
I am not aware of anything that depends on this API at the moment, other then the tests in the repo it isn't integrated with anything. In the future I think spvc in shaderc will be using it, but I checked that hasn't been implemented yet.

Its implementation is currently incomplete and hasn't really been advertised as a thing to use, since the spec it is implementing is still being worked on.